src/MainCase/MainCase.scad

```diff
@@ -236,9 +236,9 @@ module MainCase(without_inserts=false) {
           polygon(polyRound([
             [wall_thickness, wall_thickness+HeatsetInsert_height, 0],
             [OBS_height-wall_thickness, wall_thickness+HeatsetInsert_height, 0],
-            [OBS_height-wall_thickness, OBS_depth-20, 0],
```

Comment:
Clearance in this corner was required for the sensor board
![image](https://user-images.githubusercontent.com/44007906/148275155-96bd5ba7-f886-4da5-b6bb-d626e01a38eb.png)

Reply:
The heatsert inserts from the top might stick through there. Don't assume MainCase to be complete and working as of yet ;)